plugins/org.eclipse.glsp.server/src/org/eclipse/glsp/server/actionhandler/DisposeClientSessionActionHandler.java

```diff
@@ -18,19 +18,19 @@
 import java.util.List;
 
 import org.eclipse.glsp.api.action.Action;
-import org.eclipse.glsp.api.action.kind.DisposeClientAction;
+import org.eclipse.glsp.api.action.kind.DisposeClientSessionAction;
 import org.eclipse.glsp.api.model.GraphicalModelState;
 import org.eclipse.glsp.api.protocol.ClientSessionManager;
 
 import com.google.inject.Inject;
 
-public class DisposeClientActionHandler extends BasicActionHandler<DisposeClientAction> {
+public class DisposeClientSessionActionHandler extends BasicActionHandler<DisposeClientSessionAction> {
 
    @Inject()
    protected ClientSessionManager sessionManager;
 
    @Override
-   protected List<Action> executeAction(final DisposeClientAction action, final GraphicalModelState modelState) {
+   protected List<Action> executeAction(final DisposeClientSessionAction action, final GraphicalModelState modelState) {
       sessionManager.disposeClientSession(modelState.getClientId());
```

Comment:
The DisposeClientSessionAction contains the ID of the session to dispose; however it's not used here (Since the model state already contains this ID, the value should be the same; but then maybe we don't need the clientId in the DisposeClientSessionAction?)